.github/workflows/mission-control-sync-425.yml

```
@@ -0,0 +1,34 @@
+  push:
+    branches:
+      - chatgpt/finance-exit-central-cutover
+    paths:
+      - .github/workflows/mission-control-sync-425.yml
+
+permissions:
+  contents: write
+
+jobs:
+  sync-main:
+    if: github.event.head_commit.message == 'chore: seed PR 425 canonical main sync'
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)